packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -196,8 +196,9 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
   }
 }
 
-class _TooltipPositionDelegate extends SingleChildLayoutDelegate {
-  _TooltipPositionDelegate({
+/// A delegate that positions a tooltip relative to a target.
```

Comment:
("A delegate that positions a tooltip relative to a target" doesn't pass our style guide test of "the doc couldn't be written by someone who doesn't actually know anything about the class except its name", and the other members, like the constructor and fields, are lacking docs.)